blocks/toc/toc.js

```diff
@@ -78,4 +78,8 @@ export default async function decorate(block) {
   } else {
     buildTOCSide(ul, block);
   }
+
+  if (block.classList.contains('no-bottom-spacing')) {
```

Comment:
Is `no-bottom-spacing` still needed? I don't see it in styles.

Reply:
No, I'll remove this, thanks